src/test/java/com/sonyericsson/hudson/plugins/gerrit/trigger/mock/Setup.java

```diff
@@ -817,10 +817,8 @@ public static RefReplicated createRefReplicatedEvent(String project, String ref,
     public static void lockDown(JenkinsRule j) throws Exception {
         SecurityRealm securityRealm = j.createDummySecurityRealm();
         j.getInstance().setSecurityRealm(securityRealm);
-
-        GlobalMatrixAuthorizationStrategy authorizationStrategy = new GlobalMatrixAuthorizationStrategy();
-        authorizationStrategy.add(Hudson.READ, "authenticated");
-        j.getInstance().setAuthorizationStrategy(authorizationStrategy);
+        j.getInstance().setAuthorizationStrategy(
+                new MockAuthorizationStrategy().grant(Hudson.READ).everywhere().toAuthenticated());
```

Comment:
I remembered it from a code review some time back, so I knew it was available :)